client/src/components/game/phases/discardPhase/ContainerDiscard.vue

```diff
@@ -24,31 +24,20 @@
           </div>
         </div>
       </div>
-
-      <div class="chat-container">
-        <div class="section-text">
-          <p>Chat</p>
-        </div>
-        <div class="chat">
-          <Chat/>
-        </div>
-      </div>
     </div>
   </div>
 </template>
 
 <script lang="ts">
 import { Vue, Component } from 'vue-property-decorator';
-import BarDiscard from '@port-of-mars/client/components/gamedashboard/global/cards/BarDiscard.vue';
-import InventoryTable from '@port-of-mars/client/components/newGameDashboard/global/modals/InventoryTable.vue';
-import Chat from '@port-of-mars/client/components/gamedashboard/right/ChatRework.vue';
+import BarDiscard from './BarDiscard.vue';
+import InventoryTable from '@port-of-mars/client/components/reusables/InventoryTable.vue';
```

Comment:
ditto @chrstngyn concerns - remove `reusables` and move all of its contents to the `components` top level. If you need to make a new `inventory` subdirectory that's fine, but I wouldn't worry about it until you have more than one inventory related component to manage.